### PR TITLE
Fix for #21 and adding TSDoc documentation comments

### LINKED ISSUE #21: Remove extra type exports from Driftory source file?
In order to integrate this new viewer into http://www.driftory.com/, I'm needing to convert it to an old-fashioned browser script (since the Driftory website code is fairly old-fashioned, being written seven years ago). I'm using http://browserify.org/ which works great, except it chokes on the two extra type exports here:

https://github.com/pixfabrik/driftory-viewer/blob/b93f24fc231cac97e164890e2825b8f04417519e/src/library/driftory.ts#L3-L4

I'm new enough to TypeScript that I don't know the purpose of those exports. Is it possible to remove them from that file and accomplish their purpose some other way? @Dan503 any thoughts?

I imagine if I'm running into this, other people might as well.

### PR DESCRIPTION
**Pull request by Dan503**
- Fixes #21 (removed the problematic exports)
- Added some [TSDoc](https://github.com/microsoft/tsdoc) documentation comments.

The TSDoc comments allow you to see them when using the code.

For example if you hover over `onNoNext` in `demo.ts` you will see this:

image

### CODE REVIEW

**Comment by iangilman** [comment]
Awesome! Thank you on both counts! I went ahead and refined the comments further, and removed prefixUrl entirely, since it's just a leftover from other OSD stuff. Take a look at my commit on this PR for details.

**Comment by iangilman** [comment]
...And published!

File: src/library/driftory.ts [hunk 1]
```diff
@@ -44,7 +44,7 @@ export interface DriftoryArguments {
    * To prevent this, you can use this parameter to provide your own instance of OpenSeaDragon instead.
    *  */
   OpenSeadragon?: OpenSeadragonType;
-  /** Called whenever driftory navigates to a new frame (excludes when the user drags manually). */
+  /** Called whenever driftory navigates to a new frame, whether via clicking, dragging, keys, or API. */
```
Comment (Dan503) [inline src/library/driftory.ts:47]: Oh cool! I didn't realise that it also triggered on mouse drag. I thought that would have been too difficult to calculate 😊

File: src/library/driftory.ts [hunk 2]
```diff
@@ -329,7 +326,8 @@ export default class Driftory {
     }
   }
 
-  /** Get the currently active frame index (if there is one) */
+  /** Get the currently active frame index. This will be whatever frame is in the middle of the
```
Comment (Dan503) [inline src/library/driftory.ts:329]: Thanks for the extra explanation. I'm glad it also detects mouse drag 😊

Comment (iangilman) [reply src/library/driftory.ts:329]: Awesome. Thank you for getting the documentation started… Seeing how you perceive things helps me understand what needs expanding. Together we can documented better than either of us could on our own.

**Pull request merged**